src/g_ltac_iter.mlg

```diff
@@ -154,28 +158,29 @@ let pp_collection _ _ _ = WITH_DB.pr_collection
 open Pcoq.Prim
 open Pcoq.Constr
 
+}
 
 ARGUMENT EXTEND collection
-PRINTED BY pp_collection
-  | [ "db:" preident(x) ] -> [ WITH_DB.HintDb x ]
-  | [ "mod:" constr(m) ]  -> [ WITH_DB.Module m ]
-  | [ "*|-" ] -> [ WITH_DB.Premise ]
-  | [ "rev" collection(e) ] -> [ WITH_DB.Reverse e ]
-  | [ "ctors:" constr(t) ] -> [ WITH_DB.Ctors t ]
-  | [ "(" collection(e) ")" ] -> [ e ]
-END;;
+PRINTED BY { pp_collection }
+  | [ "db:" preident(x) ] -> { WITH_DB.HintDb x }
```

Comment:
Is there any way to get the lexing location here? That would allow us to print it above.

Reply:
Unfortunately, I do not have the OCaml skills needed to do this properly.